src/js/common/components/RequiredLoginView.jsx

```diff
@@ -103,7 +103,7 @@ RequiredLoginView.propTypes = {
   authRequired: PropTypes.number.isRequired,
   serviceRequired: PropTypes.string.isRequired,
   userProfile: PropTypes.object.isRequired,
-  loginUrl: PropTypes.string,
+  loginUrl: PropTypes.object.isRequired
```

Comment:
I was thinking the component that needs the props should be responsible for requiring them, and the parent shouldn't need to require the prop that will be passed down. E.g., if the child component were conditionally rendered, the parent wouldn't need to have a defined value for the prop.